CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Inhibit `MatchingNumberOfPrometheusAndCluster` when kube-state-metrics is down
```

Comment:
```suggestion
- Inhibit `MatchingNumberOfPrometheusAndCluster` when kube-state-metrics is down
  to prevent bogus pages when `kube_pod_container_status_running` metric
  isn't available
```